From 88a5d6e7487aea21bd699afbd065d4a6fd130ff0 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 17 May 2018 07:33:07 +0200
Subject: [PATCH 1/1] efi_selftest: build miniapps on x86_64

Now that we correctly define EFI_CRT0 and EFI_RELOC on x86_64 we can build
the unit tests for StartImage.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_selftest/Makefile | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/lib/efi_selftest/Makefile b/lib/efi_selftest/Makefile
index 4fe404d88d8..37608f0680b 100644
--- a/lib/efi_selftest/Makefile
+++ b/lib/efi_selftest/Makefile
@@ -39,10 +39,6 @@ ifeq ($(CONFIG_BLK)$(CONFIG_PARTITIONS),yy)
 obj-$(CONFIG_CMD_BOOTEFI_SELFTEST) += efi_selftest_block_device.o
 endif
 
-# TODO: As of v2018.01 the relocation code for the EFI application cannot
-# be built on x86_64.
-ifeq ($(CONFIG_X86_64),)
-
 ifneq ($(CONFIG_CMD_BOOTEFI_SELFTEST),)
 
 obj-y += \
@@ -68,5 +64,3 @@ $(obj)/efi_selftest_startimage_exit.o: $(obj)/efi_miniapp_file_image_exit.h
 $(obj)/efi_selftest_startimage_return.o: $(obj)/efi_miniapp_file_image_return.h
 
 endif
-
-endif
-- 
2.17.0

